test/yagna/helpers/payment.py

```diff
@@ -0,0 +1,20 @@
+"""Helper functions for easy handling of payments."""
+
+from typing import List, Tuple
+
+from goth.runner.provider import ProviderProbeWithLogSteps
+from goth.runner.requestor import RequestorProbeWithApiSteps
+
+
+async def pay_all(
+    requestor: RequestorProbeWithApiSteps,
+    agreements: List[Tuple[str, ProviderProbeWithLogSteps]],
+):
+    """Pay for all Agreements."""
+    for agreement_id, provider in agreements:
+        await provider.wait_for_invoice_sent()
+        invoices = await requestor.gather_invoices(agreement_id)
+        assert all(inv.agreement_id == agreement_id for inv in invoices)
+        # TODO:
```

Comment:
And is there anything left to do here:?
https://github.com/golemfactory/goth/blob/b3110949e7e5eb2e3e2f5eafff590057fc06939e/test/yagna/e2e/test_e2e_vm.py#L206
Because I copy-pasted it ;)